cellbase-lib/src/main/java/org/opencb/cellbase/lib/impl/core/OntologyMongoDBAdaptor.java

```diff
@@ -114,9 +116,15 @@ public Bson parseQuery(OntologyQuery query) {
         try {
             for (Map.Entry<String, Object> entry : query.toObjectMap().entrySet()) {
                 String dotNotationName = entry.getKey();
-                if (!"dataRelease".equals(dotNotationName)) {
-                    Object value = entry.getValue();
-                    createAndOrQuery(value, dotNotationName, QueryParam.Type.STRING, andBsonList);
+                switch (dotNotationName) {
+                    case "dataRelease":
```

Comment:
What if we use variables instead of hardcoding the values?

Reply:
you're right; thanks